src/test/java/com/hotels/bdp/circustrain/bigquery/client/BigQueryMetastoreClientTest.java

```diff
@@ -255,4 +269,21 @@ public void listPartitionsWithoutTableCachedAndPartitionsSizeNegativeReturnsAllP
     assertEquals(10, results.size());
     verify(cache).get(databaseName, tableName);
   }
+
+  private void setUpSchema() throws IOException {
+    byte[] content = getContentFromFileName("usa_names.avro");
+
+    when(extractionService.getStorage()).thenReturn(storage);
+    when(storage.list(anyString(), any(BlobListOption.class), any(BlobListOption.class))).thenReturn(blobs);
+    when(blobs.iterateAll()).thenReturn(Arrays.asList(blob));
+    when(blob.getContent()).thenReturn(content);
+    schema = new String(getContentFromFileName("usa_names_schema.avsc"));
+  }
+
+  private byte[] getContentFromFileName(String name) throws IOException {
```

Comment:
A better name might be `extractBytesFromFile`. You're not actually getting the bytes from the file name, it just so happens you're referring to a file by name rather than by `File`. If you make the param `name` be `fileName` then I think it's clear what the method is doing without needing that in the method name.